pycloudlib/ibm/instance.py

```diff
@@ -452,13 +452,15 @@ def __init__(
         instance: dict,
         floating_ip: Optional[dict] = None,
         username: Optional[str] = None,
+        using_existing_floating_ip: Optional[bool] = False,
```

Comment:
If the default is `False` and we do not have any meaning for `None`, why not:

```suggestion
        using_existing_floating_ip: bool = False,
```

Reply:
Good suggestion!